storage/test/authz_models.go

```diff
@@ -199,7 +200,7 @@ func ReadTypeDefinitionTest(t *testing.T, datastore storage.OpenFGADatastore) {
 			},
 		}
 
-		err := datastore.WriteAuthorizationModel(ctx, store, modelID, []*openfgapb.TypeDefinition{expectedTypeDef})
+		err := datastore.WriteAuthorizationModel(ctx, store, modelID, typesystem.SchemaVersion1_0, []*openfgapb.TypeDefinition{expectedTypeDef})
```

Comment:
is there a test that attempts to write a model with version 1_1?

Reply:
Yeah, let's add a test to write and read a model v1.0 and model v1.1 so that we can assert the deserialized schema_version string get's cast to the appropriate enum.